File: ty.py
import datetime;
import os;
from time import sleep;

def typefile(fname):
	f = open(fname)
	while True:
		l = f.readline()
		if not l: break
		print(l)
		sleep(1)
	f.close()

start = datetime.datetime(2020, 12, 5, 0, 0, 0)
while True:
	now = datetime.datetime.now()
	day = now - start 
	fname = "" + str(day.days) + ".txt"
	typefile("/home/pi/code/eng100/" + fname);
	print("*")
	sleep(3)
	os.system('clear')

File: ty.php
<?php

while (true) {
	$Day0		= "2020-12-05";
	$Day0       = strtotime($Day0);
	$DayN       = strtotime(date("Y-m-d"));
	$N      = ($DayN - $Day0)/60/60/24;

	$fname = "/home/pi/code/eng100/" . $N . ".txt";
	$handle = fopen($fname, "r");
	if ($handle) {
    	while (($line = fgets($handle)) !== false) {
			echo $line;
			sleep(3);
    	}
    	fclose($handle);
	} else {
    	// error opening the file.
	} 
	echo "*";
	sleep(10);
	system('clear');
}

?>

